app/src/common/hooks/useFhirResourceTreeData.ts

```diff
@@ -253,6 +253,24 @@ export const buildTree = (
   }
 };
 
+const findChildAttributes = (
+  attributeToDelete: Attribute,
+  attributes: Attribute[]
+): Attribute[] => {
+  const path = attributeToDelete.path;
+  return attributes.filter((attribute) => attribute.path.startsWith(path));
+};
+
+const computeChildPathIndex = (parent: ElementNode) => {
```

Comment:
return type is missing

could you add a comment describing what the function does ?